zypp/PoolItem.h

```diff
@@ -104,6 +104,9 @@ namespace zypp
 
       /** Broken (needed) but locked patches. */
       bool isUnwanted() const;
+
+      std::string statusName () const;
```

Comment:
I'd disambiguate it, so it does not get confused with the PoolItems status. And a conversion to string is usually soemthing with asString, so `patchStatusAsString` ore something like this,.

Reply:
Thought about this, but wasn't completely sure its only used for patches , will change it